themes/custom/droopler_theme/scss/components/_buttons.scss

```diff
@@ -6,16 +6,15 @@ Styleguide 2.banner.celina
 
 // Paddings and margins.
 $button-min-width: 18rem !default; // Minimum width of the button (desktop)
-$button-mobile-min-width: 16rem !default;  // Minimum width of the button (mobile)
 
 // Standard Bootstrap button
 .btn {
   @include font-button;
-  min-width: $button-min-width;
```

Comment:
po tym na mobilce szerokosc bedzie taka jak tekst, tak?

Reply:
czyli jakos tak: 
![image](https://user-images.githubusercontent.com/37533838/70447054-6501e780-1a9e-11ea-85e8-9611ebabdb2e.png)

Reply:
sprawdzałes to co ja wcześniej sugerowałem: 
![image](https://user-images.githubusercontent.com/37533838/70447126-895dc400-1a9e-11ea-8910-7a515133ff59.png)
ustawiać max-width, a width dawać na 100% ?

Reply:
wtedy chyba byłoby jakos tak... wdyt ?

![image](https://user-images.githubusercontent.com/37533838/70447205-aa261980-1a9e-11ea-973f-1404f58befa0.png)
![image](https://user-images.githubusercontent.com/37533838/70447222-b14d2780-1a9e-11ea-9cfd-dfa4c582e8d4.png)
![image](https://user-images.githubusercontent.com/37533838/70447242-b90ccc00-1a9e-11ea-93ef-590afe44b130.png)